scripts/reindex_articles.coffee

```diff
@@ -0,0 +1,22 @@
+require('node-env-file')(require('path').resolve __dirname, '../.env')
+mongojs = require 'mongojs'
+path = require 'path'
+{ indexForSearch } = Save = require '../api/apps/articles/model/save'
+Article = require '../api/apps/articles/model/index'
+search = require '../api/lib/elasticsearch'
+
+# Setup environment variables
+env = require 'node-env-file'
+switch process.env.NODE_ENV
+  when 'test' then env path.resolve __dirname, '../.env.test'
+  when 'production', 'staging' then ''
+  else env path.resolve __dirname, '../.env'
+
+# Connect to database
+db = mongojs(process.env.MONGOHQ_URL, ['articles'])
+
+db.articles.find({ }).toArray (err, articles) ->
```

Comment:
That's right this is more of a one-off job - we will run it each time we update the mappings in the elasticsearch index but that will be an infrequent occurrence. I have been able to run this fine for all our articles, though it is a slowish operation which I guess could block the db for a short period (2-5 mins).